### PR TITLE
feature/63-Get-meals-tests

### PR DESCRIPTION
**Pull request by Wojtazooo**

### CODE REVIEW

File: Backend/CateringBackendUnitTests/Handlers/MealsHandlers/GetMealsQueryHandlerTestsData.cs [hunk 1]
```diff
@@ -0,0 +1,308 @@
...
+                new() { Id = Guid.NewGuid(), Calories = 40, IsAvailable = false },
+            };
+
+            for (var calories = 0; calories <= 40; calories += 10)
+            {
+                yield return new object[]
+                {
+                    new GetMealsQuery
+                    {
+                        Calories = 10
```
Comment (Rippp) [inline Backend/CateringBackendUnitTests/Handlers/MealsHandlers/GetMealsQueryHandlerTestsData.cs:172]: change the const 10 to calories for variable

File: Backend/CateringBackendUnitTests/Handlers/MealsHandlers/GetMealsQueryHandlerTests.cs [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+        [InlineData(5, 5)]
+        [InlineData(3, 2)]
+        [InlineData(5, 2)]
+        [InlineData(2, 0)]
+        [InlineData(0, 0)]
+        public async void GivenEmptyGetMealsQuery_WhenHandle_ThenReturnsOnlyAvailableMeals(int numberOfMealsToAdd, int numberOfMealsUnavailable)
+        {
+            // Arrange
+            var mealsToSeedDatabase = new List<Meal>();
+            foreach (var i in Enumerable.Range(0, numberOfMealsToAdd))
```
Comment (Rippp) [inline Backend/CateringBackendUnitTests/Handlers/MealsHandlers/GetMealsQueryHandlerTests.cs:39]: are there any advantages of those foreach loops? they look more confusing than a simple for(;;) loop and seem over the top

File: Backend/CateringBackendUnitTests/Handlers/MealsHandlers/GetMealsQueryHandlerTestsData.cs (hunk 1 above)
Comment (Wojtazooo) [reply Backend/CateringBackendUnitTests/Handlers/MealsHandlers/GetMealsQueryHandlerTestsData.cs]: thanks for noticing that, my mistake, fixed

File: Backend/CateringBackendUnitTests/Handlers/MealsHandlers/GetMealsQueryHandlerTests.cs (hunk 2 above)
Comment (Wojtazooo) [reply Backend/CateringBackendUnitTests/Handlers/MealsHandlers/GetMealsQueryHandlerTests.cs]: changed to typical for

**Review by Rippp — APPROVED** [review]

**Pull request merged**